packages/app-project/components/ClassifyPage/ClassifyPage.spec.js

```diff
@@ -0,0 +1,20 @@
+import { shallow } from 'enzyme'
+import React from 'react'
+
+import ClassifyPage from './ClassifyPage'
+
+let wrapper
+
+describe('Component > ClassifyPage', function () {
+  before(function () {
+    wrapper = shallow(<ClassifyPage />)
+  })
+
+  it('should render without crashing', function () {})
+
+  it('should render the `FinishedForTheDay` component', function () {
+    // `FinishedForTheDay` is wrapped in a HOC, so we're just asserting on the
+    // text to include the component name.
+    expect(wrapper.text()).to.contain('FinishedForTheDay')
```

Comment:
This is a better way than I've been handling wrapped components in tests.

Reply:
I think it's okay for simple assertions on existence, but I'm still not really happy with it. If it works for you, it'll do for now.

Reply:
You have to use enzyme's dive method which can be ugly if you have to use several chains or what I've done is do an exact match on the wrapped component display name, which isn't great either. 

For MST components, You can get to the wrapped component by `InjectedWrapperComponent.wrappedComponent`. I was able to fix several broken tests the other day by using this.

Reply:
This came up the other day for me on PFE when I wrapped a component in Redux connect() and broke a bunch of tests that looked for it by display name. Turns out the most reliable way to test for a component in enzyme is to look for the constructor, so I ended up doing that.
https://github.com/zooniverse/Panoptes-Front-End/blob/04eafd6d8e5b33483b08484b3cb2966d1748a07b/app/pages/project/home/home-workflow-buttons.spec.js#L48-L53

Reply:
https://github.com/airbnb/enzyme/issues/589

Reply:
That sounds better to me; I've updated the tests.